app/src/main/java/com/example/parkingproject/utils/Picker.java

```diff
@@ -11,10 +12,9 @@
 import java.util.Date;
 
 public class Picker {
-
+    private Date date;
     private DatePickerDialog datePickerDialog;
-    static final SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yy-MM-dd HH:mm");
-    Date date;
+    private static final String FORMAT = "yy-MM-dd HH:mm";
```

Comment:
Define static members at the top of the class, general convention is organize the class as follows: interfaces/inner classes, static members, instance members, constructors and methods.